tsp-core/src/main/java/pl/edu/agh/iet/tsp/core/config/DatabaseConfig.java

```diff
@@ -0,0 +1,51 @@
+package pl.edu.agh.iet.tsp.core.config;
```

Comment:
Okej, generalnie przyświecała mi taka idea, żeby wszystko, co związane z konkretną implementacją bazy (u nas z Morphią) było zamknięte w jakiejś tam jednej, dobrze zdefiniowanej całości, którą będzie można sobie ewentualnie podmienić i dostarczyć nową jar-kę gdyby ktoś chciał zmienić bazę na przykład na jakąś relacyjną, albo po prostu na innego providera (np. Spring Data).

Wychodzi na to, że w takim razie musielibyśmy zamknąć tam wszystkie klasy domenowe, DAO (i ich implementacje) oraz DatabaseConfig. Poza tym modułem mogłaby siedzieć cała reszta (kontrolery i serwisy).

Mogę zaproponować coś takiego:
 - Do `tsp-database` idą wszystkie rzeczy związane z Morphią (DAO, ich implementacje, klasy domenowe i DatabaseConfig)
 - Tworzymy `tsp-services` (albo `tsp-database-services`) z uniwersalnymi serwisami nad bazą danych.
 - Wszystkie kontrolery (łącznie z tym moim, który teraz jest w `tsp-login`) przenosimy do odpowiednich paczek w `tsp-core` (np. `pl.edu.agh.iet.tsp.core.routing.login\database\...`).
 - Jest potrzeba wydzielania osobnych modułów dla DAO i domeny? Chyba cztery moduły to i tak dość jak na taki mały projekt :D
 - Dependencje jak poniżej.

![architektura](https://cloud.githubusercontent.com/assets/6968172/25844914/4a281ad6-34ac-11e7-9fbc-16fb64bd1831.png)

Co o tym sądzisz?